### ISSUE TITLE
Is there a way to generate everything in one file?

### ISSUE THREAD

**Issue by MrHBS** [dart-lang/dartdoc#2738]
Is there a way to generate everything in one file? Constructors, properties etc.

**Comment by jcollins-g** [comment]
No.  The `index.json` generated for the search is the only place that lists everything.
